client/components/Supplier/Supplier.tsx

```diff
@@ -1,33 +1,54 @@
-import React from 'react';
+import React, {useState} from 'react';
+import { SupplierLoginForm } from '../SupplierLoginForm/SupplierLoginForm';
 
 import './Supplier.scss';
 
 export const Supplier: React.FC = () => {
+  const adminUser = {
+    email: "admin@ad.com",
+    password: "admin123"
+  }
+
+  const [user, setUser] = useState({name: "", email:""});
+  const [error, setError] = useState("");
+
+  const Login = (details: any) => {
+    console.log(details);
+
+    if(details.email == adminUser.email && details.password == adminUser.password){
```

Comment:
Just a note as we discuss offline already, this will later be replaced with backend request for authenticating user